src/modules/bar/barNPCs.ts

```diff
@@ -1523,3 +1652,102 @@ export function octoTrip() {
     })
   }
 }
+
+
+export let GrumpyElfDialog: Dialog[] = [
+  {
+    name: 'xmasplaza-quest',
+    text: `Hey friend!  We need your help!   A big storm hit Xmas Plaza last night and caused all sorts of problems`,
+    offsetY: 24,
+  },
+  {
+    text: `It is a logistics nightmare over there.  We need your help to get back on schedule for Christmas.`,//to meet our deadline.
+    triggeredByNext: () => {
+      grumpyElf.playAnimation(`Worried_Bell  `, true, 4)
+    },
+  },
+  { 
+    text: `Will you help?  I can send you there now.`,
+    offsetY: 24,
+    isQuestion: true,
+    buttons: [
+      { label: 'YES', goToDialog: 'yes-xplaza' },
+      { label: 'NO', goToDialog: 'no-xplaza' },
+    ],
+  },
+  {
+    name: 'yes-xplaza',
+    text: "Great! I am sending you there now.  When you get there talk to the elf in charge.",
+    offsetY: 18,
+    triggeredByNext: () => {
+      grumpyElf.playAnimation('Waving_bell', true, 4)//Greet_Bell
+      grumpyElf.endInteraction()
+
+      teleportTo(CoordsEnum.XmasPlazaCoords)
+    },
+    isEndOfDialog: true,
+  },
+  {
+    name: 'no-xplaza',
+    text: "Alright then, all good! If you change your mind, head to Xmas Plaza and talk to the elf in charge.",
```

Comment:
Mention the coordinates of this xmas plaza, most won't know where that is 
-20, 80